[10] iExpenses/iExpenses/Localizations/vi.lproj/Localizable.strings

```diff
@@ -0,0 +1,31 @@
+/* 
+  Localizable.strings
+  iExpenses
+
+  Created by Joe Pham on 2022-08-07.
+  
+*/
+
+/* Main View */
+"Personal Expenses" = "Chi tiêu Cá nhân";
+"Business Expenses" = "Chi tiêu Công việc";
+"School Expenses" = "Chi tiêu Học tập";
+"Other Expenses" = "Chi tiêu Khác";
+
+"Nothing yet. Tap %@ to add more" = "Chưa có chi tiêu nào. Chạm %@ để thêm";
```

Comment:
This is great 💯 You found how to translate a sentence interpolated with a SwiftUI image 🥳